src/applications/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -233,7 +244,13 @@ class IntroductionPage extends React.Component {
 
 function mapStateToProps(state) {
   const { form, user } = state;
-  return { form, user };
+  return {
+    form,
+    user,
+    signedUpForV2Beta: createIsServiceAvailableSelector(features.allClaims)(
```

Comment:
Not really something for this PR, but we might want to rethink naming anything related to v2 as 'all claims' (or at least we should try to be consistent).